### ISSUE TITLE
Current editor is not closed if you click below freezed column

### ISSUE THREAD

**Issue by andrewQwer** [handsontable/handsontable#4246]
Hi, 

If I use `fixedColumnsLeft` option then clicking below the freezed column doesn't close current editor.
JSFiddle: http://jsfiddle.net/andrewQwer/o4rwf84j/6/
Video: https://www.screencast.com/t/MUopAw80PcHe

**Comment by AMBudnik** [comment]
Thank you for sharing this issue @andrewQwer 

I am marking this as s bug.
ps. It may be related to the extra height of the `ht_clone_left` element. If we click outside the element's area we are able to close editor.

![image](https://cloud.githubusercontent.com/assets/10757813/26099914/8785503a-3a2c-11e7-9959-52ce650e62a3.png)

**Comment by andrewQwer** [comment]
Yes, `ht_clone_left` either should match total rows height or should close editor on click.

**Comment by AMBudnik** [comment]
The issue is replicable in v `2.0.0.` and doesn't seem to work fine in any of the previous versions.

**Comment by pnowak** [comment]
Good news! This issue got solved v8.0.0. Here's an updated demo - https://jsfiddle.net/v0oxqa37/

**Comment by AMBudnik** [comment]
Great @pnowak 

@andrewQwer please let us know if it works for you as well.

**Comment by andrewQwer** [comment]
Hello :) 

Thank you for the fix, but it doesn't work when `fixedColumnsLeft` is set to 1. In your example this property is commented and if you uncomment it you'll see that when you click belowe the first (freezed) column when editor is active - nothing happens.

**Comment by budnix** [comment]
Fixed in the latest Handsontable version.
